python_modules/libraries/dagster-components/dagster_components/core/component_rendering.py

```diff
@@ -14,6 +14,9 @@
 class TemplatedValueResolver:
     context: Mapping[str, Any]
 
+    def with_context(self, **kwargs) -> "TemplatedValueResolver":
+        return TemplatedValueResolver(context={**self.context, **kwargs})
+
```

Comment:
Could you either name the `kwargs` argument or not use `**` at all. It's fairly ambiguous via inspection of the declaration what is going on here.